server/routes/plan-history/PlanHistoryController.ts

```diff
@@ -13,6 +13,10 @@ export default class PlanHistoryController {
       const notes = await req.services.planService.getNotes(planUuid)
       const readWrite = req.services.sessionService.getAccessMode() === AccessMode.READ_WRITE
 
+      if (notes.length === 0) {
+        return next(HttpError(403, 'Plan has not been agreed'))
```

Comment:
Might be worth knowing that the error message is actually only displayed in development builds (specifically as part of the stack trace print).